e2e/plugin-coverage-e2e/tests/collect.e2e.test.ts

```diff
@@ -2,18 +2,25 @@ import { cp } from 'node:fs/promises';
 import { join } from 'node:path';
 import { afterAll, afterEach, beforeAll } from 'vitest';
 import { type Report, reportSchema } from '@code-pushup/models';
+import { nxTargetProject } from '@code-pushup/test-nx-utils';
 import { teardownTestFolder } from '@code-pushup/test-setup';
-import { omitVariableReportData } from '@code-pushup/test-utils';
+import {
+  E2E_ENVIRONMENTS_DIR,
+  TEST_OUTPUT_DIR,
+  omitVariableReportData,
+} from '@code-pushup/test-utils';
 import { executeProcess, readJsonFile } from '@code-pushup/utils';
 
-describe('collect report with coverage-plugin NPM package', () => {
-  const envRoot = 'tmp/e2e/plugin-coverage-e2e';
-  const fixtureDir = join('e2e', 'plugin-coverage-e2e', 'mocks', 'fixtures');
-  const basicDir = join(envRoot, 'basic-setup');
-  const existingDir = join(envRoot, 'existing-report');
+describe('PLUGIN collect report with coverage-plugin NPM package', () => {
```

Comment:
Renaming the `describe` block means there is now a new snapshot, but the old one wasn't deleted. Not sure what's the benefit of prefixing it with `PLUGIN `. Either way, please remove obsolete snapshots with `npx nx nxv plugin-coverage-e2e -- --update`.

Same goes for `plugin-eslint-e2e` and `plugin-lighthouse-e2e`.